Dockerfile

```diff
@@ -1,23 +1,22 @@
-FROM registry.access.redhat.com/ubi9/ubi-minimal:9.5-1741850109@sha256:bafd57451de2daa71ed301b277d49bd120b474ed438367f087eac0b885a668dc as prod
+FROM registry.access.redhat.com/ubi9/ubi-minimal:9.5-1741850109@sha256:bafd57451de2daa71ed301b277d49bd120b474ed438367f087eac0b885a668dc AS prod
 
 WORKDIR /schemas
 
 COPY schemas schemas
 COPY graphql-schemas graphql-schemas
 COPY LICENSE /licenses/LICENSE
 
-FROM prod as test
+FROM prod AS test
 
 RUN microdnf upgrade -y && \
-    microdnf install -y \
-        python3.11 && \
+    microdnf install -y python3.11 && \
     microdnf clean all && \
     update-alternatives --install /usr/bin/python3 python /usr/bin/python3.11 1 && \
     ln -snf /usr/bin/python3.11 /usr/bin/python && \
     python3 -m ensurepip
 
 RUN python3 -m pip install --no-cache-dir --upgrade pip tox
 
-COPY tox.ini test /schemas/
+COPY tox.ini test .yamllint /schemas/
 
 CMD ["tox"]
```

Comment:
we have `tox` in this repo? Set it in `CMD` won't actually run it in pipeline. 

We don't need `test` at all, can just use qontract validator to do bundle validation. We need to get rid of tox and reimplement `make test bundle validate gql_validate` in Dockerfile.

Reply:
Good idea. I'll tackle that in a future PR. https://issues.redhat.com/browse/APPSRE-12422